hardware/products/ucm/g400.md

```diff
@@ -1,5 +1,250 @@
 # G400
 ![G400S](images/g400s.jpg) ![G400D](images/g400d.jpg)
+# Overview
+---
+The G400-D System on Module (SoM) is the precursor to the Universal Compute Modules (UCM). It pioneered the SO-DIMM System on Module (SoM) originally running .Net Micro Framework exclusively. It now can operate various other platforms, including TinyCLR.
+
+# Specifications
+---
+|                |                                              |
+| :------------- | :------------------------------------------- |
+| Processor      | AT91SAM9X35 (Atmel/Microchip 32-bit ARM 926) |
```

Comment:
Separate entries for processor core and manufacturer will probably read better

Reply:
I don't think the columns should get a name. This is only supposed to be a list. Because of the limitations with Markdown, the table needs to be formatted this way or the table will not show. Unfortunately, it causes a blank row. I would not know what to call the first column because the information in it is not specific, only the second column is specific to the first.